docker/entrypoint.sh

```diff
@@ -49,7 +49,7 @@ _main() {
       celery --app featurebyte.worker.start.celery worker -Q io_task,io_task:1,io_task:2,io_task:3 --loglevel=INFO --pool=gevent -c 1000
     elif [ "$1" = 'server' ]; then
       python /scripts/migration.py
-      uvicorn featurebyte.app:app --host=$API_HOST --port=$API_PORT --timeout-keep-alive=300 --log-level=$LOG_LEVEL
+      uvicorn featurebyte.app:app --host=$API_HOST --port=$API_PORT --timeout-keep-alive=300 --log-level=info
```

Comment:
Do we want to fix the log level for uvicorn?

Reply:
Is it useful to be able to override the log level for uvicorn? If so, I think we can add a different variable to control that. Same for workers' log level which is currently also fixed as INFO.

Reply:
I guess not. Let's leave it as info then.